thorlcr/graph/thgraphslave.cpp

```diff
@@ -1330,6 +1331,15 @@ void CSlaveGraph::serializeDone(MemoryBuffer &mb)
         }
     }
     mb.writeDirect(cPos, sizeof(count), &count);
+
+    if (!owner)
+        queryJob().queryTempHandler()->serializeUsageStats(mb, gid);
+    else
+    {
+        IGraphTempHandler *tempHandler = queryTempHandler(false);
+        if (tempHandler)
+            tempHandler->serializeUsageStats(mb, gid);
```

Comment:
What happens if there is no tempHandler?  No stats will be serialized, and the master will try to read data that isn't there?

Reply:
good point, missed that. Need to serialize dummy values.